front/app/modules/commercial/verification/citizen/components/VerificationSignUpSteps.tsx

```diff
@@ -23,8 +23,8 @@ const VerificationSignUpSteps = ({
       isEnabled: (_, metaData) => !!metaData.verification,
       isActive: (authUser, metaData) => {
         if (isNilOrError(authUser)) return false;
-        const verificationFlow = !!metaData.verification;
-        return verificationFlow && !authUser.attributes.verified;
+        const hasVerificationStep = !!metaData.verification;
```

Comment:
Minor terminology nitpick: `hasVerificationStep` is a better description, but `metaData.verification` is specific to the current flow. Maybe `flowHasVerificationStep`?